mesonbuild/backend/ninjabackend.py

```diff
@@ -1275,8 +1275,8 @@ def generate_dynamic_link_rules(self, outfile):
                 else:
                     command_template = ' command = %s %s $ARGS  %s $in $LINK_ARGS $aliasing\n'
                 command = command_template % \
-                (' '.join(compiler.get_linker_exelist()),\
-                 ' '.join(cross_args),\
+                (' '.join(compiler.get_linker_exelist()),
```

Comment:
Put opening parenthesis on the line above and indent these by four spaces - no need for the backslash that way.